selfservice/strategy/oidc/strategy.go

```diff
@@ -692,6 +667,70 @@ func (s *Strategy) handleError(w http.ResponseWriter, r *http.Request, f flow.Fl
 	return err
 }
 
+func (s *Strategy) populateAccountLinkingUI(ctx context.Context, lf *login.Flow, usedProviderID string, duplicateIdentifier string, availableCredentials []string) {
```

Comment:
Can we also have a button to cancel the account linking? This could just redirect to the login UI, which starts a fresh flow.

Reply:
Makes sense to me!

Reply:
We can, but it's a bit more annoying to get looking good.

Either button hell:

<img width="483" alt="image" src="https://github.com/user-attachments/assets/b8a34d48-d23a-47a4-855e-680a70e39f34">

or (here the "cancel" link just kinda disappears in the form, IMO.)

<img width="488" alt="image" src="https://github.com/user-attachments/assets/7ce2c4cd-d183-4fbe-b1f5-c3b6140b1b89">

Reply:
The first variant looks good enough. May I suggest changes to the copy:

```patch
-Sign in with GitHub and link credential
+Link with GitHub account
```

Reply:
"Link" is so technical. 

"Confirm with GitHub", "Confirm with code" (though that's also non-obvious) would be better, IMO.

Reply:
The button is problematic, because typically a button does a form submit, which would mean that we would need to cancel the linking on the server side and redirect from there, causing an unnecessary redirect. 

I think the link is good, if it's ordered correctly.

Reply:
And actually rendering the cancel button is super complicated in Ory Elements, because if a user has both the password, and code method (or passkey/webauthn) configured we would show the cancel button for each group.

Untangling that, would be complicated.

--

Adding it to the card footer in ory elements if the form is an account linking form directly is much easier: 

<img width="459" alt="image" src="https://github.com/user-attachments/assets/7dd2e7cc-43a7-4129-9003-04d261ba82d1">